src/main/java/com/group6/harmoniq/controllers/SpotifyController.java

```diff
@@ -0,0 +1,157 @@
+
+// Copyright 2012-2016 Spotify AB
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+//     http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//Original Source: https://github.com/spotify/web-api-examples
+//This file was modified from the original source by Christian Chidiac on June 30, 2024.
+//This code was converted from JavaScript to Java by Christian Chidiac on June 30, 2024.
```

Comment:
I think this can be removed from here since we already added the License to the repo and we have the history of changes on each file via git